tensorflow/tools/dockerfiles/spec.yml

```diff
@@ -38,6 +38,18 @@ releases:
     versioned:
         tag_specs:
             - "{_TAG_PREFIX}{ubuntu}{jupyter}"
+            - "{_TAG_PREFIX}{ubuntu-1604-onednn}"
```

Comment:
The "versioned" section is for the stuff the TF team publishes for each release, can you remove them? You can make another "onednn" release if you just want to keep them all grouped, but you can also use the CLI to build anything in the dockerfiles section named `.*onednn.*`.

Reply:
Done. Please see my comment below.